src/Type.js

```diff
@@ -53,7 +62,6 @@ export default class Type {
 	/**
 	 * Map a number to the internal representation
 	 * @param {number} number
-	 * @returns
```

Comment:
We should update this, not remove it! Same for the `@returns` below. Happy to leave it intact as well, so that someone else can update it.